ledger/common/build.gradle

```diff
@@ -26,3 +26,6 @@ dependencies {
     testImplementation "org.assertj:assertj-core:$assertjVersion"
 }
 
+jar {
+    baseName 'corda-ledger-common'
```

Comment:
baseName is [Deprecated ](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.bundling.Jar.html#org.gradle.api.tasks.bundling.Jar:baseName) in favor of archiveBaseName .

Reply:
Also, we want to configure tasks lazily, so:
```gradle
tasks.named('jar', Jar) {
    archiveBaseName = 'corda-ledger-common'`
   ...
}
```

Reply:
Thanks. I have changed them.